src/vector_v4.hpp

```diff
@@ -109,6 +109,7 @@ template <typename T> class VectorV4 {
 
```

Comment:
pop_back() is wrong.

Did you test out the pop_back with T = struct with ctr/dtr counters ?